sysutils/dcfldd/Portfile

```diff
@@ -1,29 +1,31 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 
-name                dcfldd
-version             1.3.4-1
-revision            1
-checksums           rmd160  3a6fba197686da8f1890d21f75b6bc2ae18592a7 \
-                    sha256  f5143a184da56fd5ac729d6d8cbcf9f5da8e1cf4604aa9fb97c59553b7e6d5f8 \
-                    size    163141
+github.setup        ressurecting-open-source-projects dcfldd 1.9.1 v
+github.tarball_from releases
 
+name                dcfldd
+revision            0
 categories          sysutils
-maintainers         nomaintainer
+maintainers         {@davidpolverari gmail.com:davidpolverari} \
+                    openmaintainer
+description         enhanced version of dd for forensics and security
+long_description    ${name} is an enhanced version of the dd command with \
+                    additional features useful for forensics and information \
+                    security, designed to run on Unix or Unix-like \
+                    (POSIX-compliant) operating systems.
+
+homepage            https://github.com/resurrecting-open-source-projects/dcfldd
 license             GPL-2+
 
-description         Enhanced dd with md5 checksums
-long_description    Enhanced dd with md5 checksums
-
-homepage            http://dcfldd.sourceforge.net/
-platforms           darwin
-
-master_sites        sourceforge
-
-configure.args  --mandir=${prefix}/share/man
+fetch.type          git
+git.url             https://github.com/resurrecting-open-source-projects/dcfldd.git
```

Comment:
shouldn't be needed

Reply:
Without `git.url`, a prompt asks for GitHub username and password during fetching. And that only when `fetch.type git` is present, otherwise it tries to download distfiles from `*.distfiles.macports.org`.

Reply:
Done by @herbygillot. Thanks!